diff --git a/todo-app/docker-compose.dev.yml b/todo-app/docker-compose.dev.yml
index eb1832e..ab7b122 100644
--- a/todo-app/docker-compose.dev.yml
+++ b/todo-app/docker-compose.dev.yml
@@ -6,9 +6,10 @@ services:
       dockerfile: ./todo-frontend/dev.Dockerfile # This will simply tell which dockerfile to read
     volumes:
       - ./todo-frontend/:/usr/src/app/todo-frontend # The path can be relative, so ./ is enough to say "the same location as the docker-compose.yml"
+    ports:
+      - 5173:5173
     container_name: hello-front-dev # This will name the container hello-front-dev
-    environment:
-      - VITE_BACKEND_URL=http://localhost:8080/api
+    
   debug-helper:
     image: busybox
   server:
@@ -18,6 +19,8 @@ services:
       dockerfile: ./todo-backend/dev.Dockerfile # This will simply tell which dockerfile to read
     volumes:
       - ./todo-backend/:/usr/src/app # The path can be relative, so ./ is enough to say "the same location as the docker-compose.yml"
+    ports:
+      - 3000:3000
     container_name: todo-server # This will name the container hello-front-dev
     environment: 
       - MONGO_URL=mongodb://the_username:the_password@mongodb:27017/the_database
diff --git a/todo-app/nginx.dev.conf b/todo-app/nginx.dev.conf
index a4f5fd1..c72efa4 100644
--- a/todo-app/nginx.dev.conf
+++ b/todo-app/nginx.dev.conf
@@ -17,11 +17,7 @@ http {
       proxy_pass http://app:5173;
     }
     location /api/ {
-      proxy_http_version 1.1;
-      proxy_set_header Upgrade $http_upgrade;
-      proxy_set_header Connection 'upgrade';
-
-      proxy_pass http://server:3000/;
+      proxy_pass http://todo-server:3000
     }
   }
 }
\ No newline at end of file
diff --git a/todo-app/todo-backend/redis_data/appendonlydir/appendonly.aof.1.incr.aof b/todo-app/todo-backend/redis_data/appendonlydir/appendonly.aof.1.incr.aof
index dabec75..e99dffa 100644
--- a/todo-app/todo-backend/redis_data/appendonlydir/appendonly.aof.1.incr.aof
+++ b/todo-app/todo-backend/redis_data/appendonlydir/appendonly.aof.1.incr.aof
@@ -1,203 +1,191 @@
-*2
-$6
-SELECT
-$1
-0
-*3
-$3
-set
-$5
-count
-$1
-1
-*3
-$3
-set
-$11
-added_todos
-$1
-0
-*3
-$3
-set
-$11
-added_todos
-$2
-01
-*3
-$3
-set
-$11
-added_todos
-$3
-011
-*3
-$3
-set
-$11
-added_todos
-$2
-12
-*3
-$3
-set
-$11
-added_todos
-$1
-0
-*3
-$3
-set
-$11
-added_todos
-$1
-0
-*3
-$3
-set
-$11
-added_todos
-$1
-1
-*2
-$6
-SELECT
-$1
-0
-*2
-$4
-INCR
-$9
-mycounter
-*3
-$3
-SET
-$11
-added_todos
-$4
-9001
-*3
-$3
-set
-$11
-added_todos
-$4
-9002
-*3
-$3
-DEL
-$5
-count
-$9
-mycounter
-*2
-$3
-DEL
-$11
-added_todos
-*3
-$3
-set
-$11
-added_todos
-$1
-1
-*2
-$6
-SELECT
-$1
-0
-*3
-$3
-set
-$11
-added_todos
-$1
-2
-*3
-$3
-set
-$11
-added_todos
-$1
-3
-*2
-$6
-SELECT
-$1
-0
-*3
-$3
-set
-$11
-added_todos
-$1
-4
-*3
-$3
-set
-$11
-added_todos
-$1
-5
-*3
-$3
-set
-$11
-added_todos
-$1
-6
-*3
-$3
-set
-$11
-added_todos
-$1
-7
-*3
-$3
-set
-$11
-added_todos
-$1
-8
-*3
-$3
-set
-$11
-added_todos
-$1
-9
-*3
-$3
-set
-$11
-added_todos
-$2
-10
-*3
-$3
-set
-$11
-added_todos
-$2
-11
-*3
-$3
-set
-$11
-added_todos
-$2
-12
-*2
-$6
-SELECT
-$1
-0
-*3
-$3
-set
-$11
-added_todos
-$2
-13
+*2
+$6
+SELECT
+$1
+0
+*3
+$3
+set
+$5
+count
+$1
+1
+*3
+$3
+set
+$11
+added_todos
+$1
+0
+*3
+$3
+set
+$11
+added_todos
+$2
+01
+*3
+$3
+set
+$11
+added_todos
+$3
+011
+*3
+$3
+set
+$11
+added_todos
+$2
+12
+*3
+$3
+set
+$11
+added_todos
+$1
+0
+*3
+$3
+set
+$11
+added_todos
+$1
+0
+*3
+$3
+set
+$11
+added_todos
+$1
+1
+*2
+$6
+SELECT
+$1
+0
+*2
+$4
+INCR
+$9
+mycounter
+*3
+$3
+SET
+$11
+added_todos
+$4
+9001
+*3
+$3
+set
+$11
+added_todos
+$4
+9002
+*3
+$3
+DEL
+$5
+count
+$9
+mycounter
+*2
+$3
+DEL
+$11
+added_todos
+*3
+$3
+set
+$11
+added_todos
+$1
+1
+*2
+$6
+SELECT
+$1
+0
+*3
+$3
+set
+$11
+added_todos
+$1
+2
+*3
+$3
+set
+$11
+added_todos
+$1
+3
+*2
+$6
+SELECT
+$1
+0
+*3
+$3
+set
+$11
+added_todos
+$1
+4
+*3
+$3
+set
+$11
+added_todos
+$1
+5
+*3
+$3
+set
+$11
+added_todos
+$1
+6
+*3
+$3
+set
+$11
+added_todos
+$1
+7
+*3
+$3
+set
+$11
+added_todos
+$1
+8
+*3
+$3
+set
+$11
+added_todos
+$1
+9
+*3
+$3
+set
+$11
+added_todos
+$2
+10
+*3
+$3
+set
+$11
+added_todos
+$2
+11
+*3
+$3
+set
+$11
+added_todos
+$2
+12
diff --git a/todo-app/todo-backend/redis_data/dump.rdb b/todo-app/todo-backend/redis_data/dump.rdb
index 801437a..9756309 100644
Binary files a/todo-app/todo-backend/redis_data/dump.rdb and b/todo-app/todo-backend/redis_data/dump.rdb differ
diff --git a/todo-app/todo-frontend/src/Todos/TodoView.jsx b/todo-app/todo-frontend/src/Todos/TodoView.jsx
index 0d22bdd..d05ad9b 100644
--- a/todo-app/todo-frontend/src/Todos/TodoView.jsx
+++ b/todo-app/todo-frontend/src/Todos/TodoView.jsx
@@ -36,7 +36,7 @@ const TodoView = () => {
 
   return (
     <>
-      <h1>Todos what is going on?</h1>
+      <h1>Todos I do not know what is going on?</h1>
       <Form createTodo={createTodo} />
       <List todos={todos} deleteTodo={deleteTodo} completeTodo={completeTodo} />
     </>
